src/app/components/SignBottom.tsx

```diff
@@ -6,14 +6,16 @@ import { translateRaw } from '@translations';
 export const SignBottom = ({
   disabled,
   handleAccept,
-  handleDeny
+  handleDeny,
+  form
 }: {
   disabled: boolean;
   handleAccept?(): void;
   handleDeny(): void;
+  form?: string;
 }) => (
   <PanelBottom>
-    <Button id="accept_button" type="submit" disabled={disabled} onClick={handleAccept}>
+    <Button id="accept_button" type="submit" form={form} disabled={disabled} onClick={handleAccept}>
```

Comment:
Why?

Reply:
This makes the button submit `form`, even though it´s not directly inside that form